ark/phenotyping/som_utils.py

```diff
@@ -1337,8 +1526,16 @@ def pixel_consensus_cluster(fovs, channels, base_dir, max_k=20, cap=3,
         os.path.join(base_dir, clust_to_meta_name)
     ).astype(np.int64)
 
-    # read in the channel-averaged results across all pixel SOM clusters
-    pixel_channel_avg_som_cluster = pd.read_csv(som_cluster_avg_path)
+    # because FOVs may become corrupted, we also need to re-compute average channel expression
```

Comment:
I think it's fine to use the previous version, 1 or 2 corrupted FOVs is not going to dramatically change this

Reply:
True, when re-running the meta cluster remap I noticed that only the pixel count barplot raw numbers changed slightly. The heatmaps didn't look different.